### PR TITLE
Add role option when starting dkeyserver

### PR DESCRIPTION
**Pull request by wanghouqi**
Signed-off-by: wanghouqi <houqix.wang@intel.com>

### CODE REVIEW

File: docker/.env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # host ip
-HOST_IP=1.2.3.4 # you must be adjust 1.2.3.4 to your host IP.
+HOST_IP=10.112.240.122 # you must be adjust 1.2.3.4 to your host IP.
```
Comment (syan10) [inline docker/.env:2]: revert

File: docker/.env [hunk 2]
```diff
@@ -30,13 +30,17 @@ IMAGE_NAME_EHSM_BASE=ehsm_base
 EHSM_VERSION_STR=main # you can change the main branch to the specific tag whit you want.
 
 # pccs url
-PCCS_URL=https://1.2.3.4:8081  # you must be adjust 1.2.3.4 to your pccs server url.
+PCCS_URL=https://10.112.240.166:8081  # you must be adjust 1.2.3.4 to your pccs server url.
```
Comment (syan10) [inline docker/.env:33]: revert

File: docker/.env [hunk 3]
```diff
@@ -30,13 +30,17 @@ IMAGE_NAME_EHSM_BASE=ehsm_base
...
 
 # Domain key server port
 DKEYSERVER_PORT=8888
 
 # domain key cache socket localtion
 DKEYCACHE_SOCKET_LOCALTION='/var/run/ehsm'
+# The role of dkeyserver has two types: root and worker
+# root: It's root node server, It will create DomainKey
+# worker: It's worker node server, It must be get a DomainKey from other dkeyserver.
```
Comment (syan10) [inline docker/.env:42]: update it based on my reply in the chat box

**Comment by yang8621** [comment]
title: dokcer -> docker

File: docker/.env [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # host ip
-HOST_IP=1.2.3.4 # you must be adjust 1.2.3.4 to your host IP.
```
Comment (syan10) [inline docker/.env:2]: commit title:
"
Add role option when starting dkeyserver
"

File: docker/.env (hunk 2 above)
Comment (wanghouqi) [reply docker/.env]: fix

File: docker/.env (hunk 3 above)
Comment (wanghouqi) [reply docker/.env]: fix

File: docker/.env (hunk 4 above)
Comment (wanghouqi) [reply docker/.env]: fix

**Comment by wanghouqi** [comment]
> title: dokcer -> docker

fix

**Review by syan10 — APPROVED** [review]

**Pull request merged**